test/alphabet/nucleotide/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 add_executable(dna5_test dna5_test.cpp)
```

Comment:
...

Okay then lets argue:

* The intention and wording of this PR is `unify all test case names and tidy up test cases`, it clearly states ALL test cases, not some, not one nor two, but all

* This PR is a complete and self-contained unit which fixes all inconsistencies at this moment.

* My changes are not a 1-1 copy of your PR. I applied the rules I set for the test cases. So there might be differences between your pull request and mine. Those will only surface if we have a proper merge conflict. If your pull request gets accepted before mine, I will fix them and if mine gets accepted before yours, you will have to fix it.

* But, it makes no sense to have a PR that is in a state in-between, because theoretically someone could use those faulty excluded files and propagate the wrong existing scheme. (The most just copy and paste and change accordingly)

* Furthermore, my PR has zero in common with yours, from a purist perspective your changes to CMakeLists.txt have nothing to do with the intention of your PR. I know this is a stupid argument :smile: 

* And now the point why I jokingly said "are you afraid of a simple merge conflict?". For you it only means that you discard all your changes and copy and paste the test cases that are new. And I don't want to loose this completeness factor only because of this small merge conflict.